### PR TITLE
FEATURE: Display current db name in logs when running multisite.

### PR DESCRIPTION
**Pull request by tgxworld**

### CODE REVIEW

File: lib/rails_multisite/formatter.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+module RailsMultisite
+  class Formatter < ::ActiveSupport::Logger::SimpleFormatter
+    def call(severity, timestamp, progname, msg)
+      string = super
+
+      if RailsMultisite::ConnectionManagement.all_dbs != [RailsMultisite::ConnectionManagement::DEFAULT]
```
Comment (SamSaffron) [inline lib/rails_multisite/formatter.rb]: why introduce the formatter at all if its not running multisite? no need to have the check and make arrays each time, only introduce the formatter if it truly is multisite.

**Comment by tgxworld** [comment]
@SamSaffron Updated. I missed the check for a config file.

**Comment by tgxworld** [comment]
ping @SamSaffron  :grin:

**Comment by SamSaffron** [comment]
looks good, thanks

**Pull request merged**